macros/edr/data_monitoring/monitors_query/column_monitoring_query.sql

```diff
@@ -24,7 +23,7 @@
         {%- else %}
             true
         {%- endif %}
-        {% if where_expression %} and {{ where_expression }} {% endif %}
+        {% if metric_properties['where_expression'] != "Null" %} and {{ metric_properties['where_expression'] }} {% endif %}
```

Comment:
What do you think about leaving it as `null` and keeping the same comparison?

Reply:
problem is, json_serializing a dict with a None value doesn't work

Reply:
https://github.com/elementary-data/dbt-data-reliability/pull/246#discussion_r1100007409

Reply:
closing per above discussion